package.json

```diff
@@ -151,12 +142,12 @@
 		"fixtures:generate": "npm run fixtures:server-registered && cross-env GENERATE_MISSING_FIXTURES=y npm run test-unit",
 		"fixtures:regenerate": "npm run fixtures:clean && npm run fixtures:generate",
 		"package-plugin": "./bin/build-plugin-zip.sh",
-		"test-unit": "jest",
+		"test-unit": "wp-scripts test",
```

Comment:
I guess this is more a package issue, but I wonder if this command should be `wp-scripts unit-test` to allow for other kind of test commands.

Reply:
In the future, we might have also e2e support but we are very far away from this :) We can rename it or add a flag like `--type=unit` or something along those lines. At the moment it's a simple wrapper for `jest`.